.github/CHANGELOG.md

```diff
@@ -19,12 +19,15 @@
   can now be changed after device creation.
   [#317](https://github.com/XanaduAI/pennylane/pull/317)
 
-### Documentation
-
-* Unify import shortcuts to be under qml in qnode.py
+* Unified import shortcuts to be under qml in qnode.py
```

Comment:
(Note: This change appears here as it fixes an incorrect merge conflict from a previous PR)